### DISCUSSION TITLE
Adding relationship to existing ones

### DISCUSSION THREAD

**Discussion by bjv** [Budibase/budibase#6126] [Help] [answered]
Hej Guys,

what transform expression could be used if a newly saved row should be added to an existing 1:n relationship colum.
Like "Save Row" with:
![image](https://user-images.githubusercontent.com/1166771/171107546-ac91d646-cd68-488d-96fc-e394601d6c5e.png)

This works; EXCEPT this expression replaces the existing relationship in "HelpdeskTickets" by just the newly created Row-Relationship. I'd rather to add this to the existing relationships, like:

```
old_relationships  = {{ Invest Repeater.Invests.HelpdeskTickets }}
return old_relationships.append({{ Action 1.Saved row._id }})
// I know this is no valid code; just my own style pseudo code for clarification
```
(How) Can this be realized?

**Comment by melohagan** [comment, marked as answer]
Hey @bjv 

Your pseudo code isn't that far off.

**JavaScript Binding**
```
let oldRelationships = $("Invest Repeater.Invests.HelpdeskTickets"); //this binding should be selectable on the left hand pane.
return oldRelationships.concat([$("Action 1.Saved row._id")]);
```

**Reply by bjv** [reply]
Hej @melohagan
Works perfectly. Thanks a lot!
